Core/include/Acts/Vertexing/AdaptiveMultiVertexFitter.hpp

```diff
@@ -98,6 +104,23 @@ class AdaptiveMultiVertexFitter {
         }
       }
     }
+
+    Result<void> removeVertexFromCollection(Vertex<InputTrack_t>& vtxToRemove) {
```

Comment:
I see this is not the first one, but I think we should avoid methods in state structs if possible.

Reply:
Will try to move them to the main class